functions/.eslintrc.js

```diff
@@ -36,7 +36,9 @@ module.exports = {
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-unsafe-return': 'off',
     '@typescript-eslint/no-var-requires': 'off',
+    '@typescript-eslint/restrict-template-expressions': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/ban-types': 'off',
```

Comment:
Which type did you had to ban?

Reply:
[This one](https://github.com/meilisearch/firestore-meilisearch/blob/logger/functions/src/logs.ts#L53):
```
export function addDocument(id: string, data: object) {
```
And also [this](https://github.com/meilisearch/firestore-meilisearch/blob/logger/functions/src/logs.ts#L65) and [this](https://github.com/meilisearch/firestore-meilisearch/blob/logger/functions/src/logs.ts#L76) for the same reasons.
I had this eslint warning/error
```
Don't use `object` as a type. The `object` type is currently hard to use ([see this issue](https://github.com/microsoft/TypeScript/issues/21732)).
Consider using `Record<string, unknown>` instead, as it allows you to more easily inspect and use the keys.eslint@typescript-eslint/ban-types
```

Reply:
why dont you use `Record<string, unknown>` ?

Reply:
data could be the following: 
```js
type Document = {
  [key: string]: any
}
```

and thus documents: `[Document]`